Makefile

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
I'm down with the Makefile

Esp when it comes to more complicated scripts, writing them in package.json starts to become a pain